<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickSSLClass</class>
 <widget class="QMainWindow" name="QuickSSLClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuickSSL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="SHA">
       <attribute name="title">
        <string>SHA</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="sha">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="sha1">
           <attribute name="title">
            <string>SHA-1</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPlainTextEdit" name="sha1_source"/>
                </item>
                <item>
                 <widget class="QPlainTextEdit" name="sha1_encoded"/>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="sha1_encode">
                <property name="text">
                 <string>Encode!</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="sha256">
           <attribute name="title">
            <string>SHA-256</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPlainTextEdit" name="sha256_source"/>
                </item>
                <item>
                 <widget class="QPlainTextEdit" name="sha256_encoded"/>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="sha256_encode">
                <property name="text">
                 <string>Encode!</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="sha512">
           <attribute name="title">
            <string>SHA-512</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPlainTextEdit" name="sha512_source"/>
                </item>
                <item>
                 <widget class="QPlainTextEdit" name="sha512_encoded"/>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="sha512_encode">
                <property name="text">
                 <string>Encode!</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="MD5">
       <attribute name="title">
        <string>MD5</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="md5_encode">
              <property name="text">
               <string>Encode!</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPlainTextEdit" name="md5_source"/>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="md5_encoded"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="RSA">
       <attribute name="title">
        <string>RSA</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QTabWidget" name="rsa">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="gen">
           <attribute name="title">
            <string>RSA generate</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="bits">
                <property name="text">
                 <string>Bits:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="bitsettings">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>1024 bits (fastest but not too safe)</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2048 bits (fast and safe) (recommended)</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>4096 bits (safest but slowest)</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="rsa_gen">
              <property name="text">
               <string>Generate Private &amp;&amp; Public Key</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="crypto">
           <attribute name="title">
            <string>RSA encrypt/decrypt</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The author is so lazy that he didn't even keep a line of code.&lt;/p&gt;&lt;p&gt;(In fact, he didn't know how to do it.)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="base64">
       <attribute name="title">
        <string>Base64 convert</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="1" column="1">
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="base64_encode">
              <property name="text">
               <string>Encode! (--&gt;)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="base64_decode">
              <property name="text">
               <string>Decode! (&lt;--)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPlainTextEdit" name="base64_source"/>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="base64_encoded"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings">
       <attribute name="title">
        <string>settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Language:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="locale"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="save">
            <property name="text">
             <string>save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="quickssl.qrc"/>
 </resources>
 <connections/>
</ui>
